pom.xml

```diff
@@ -27,21 +27,32 @@
                 </configuration>
             </plugin>
             <plugin><!-- run unit test in src/test/java -->
-                <groupId>org.codehaus.gmaven</groupId>
-                <artifactId>gmaven-plugin</artifactId>
+                <groupId>org.codehaus.gmavenplus</groupId>
```

Comment:
i have to look into this eventually, but what does gmaven plus do?

Reply:
It’s a groovy compiler, successor to gmaven, Java / groovy interop wasn’t working fully without using the more modern plus rewrite. 

I wanted to reuse the code that checks a user has logged into Jira and Artifactory